libraries/lib-datahandler/src/main/thrift/projects.thrift

```diff
@@ -14,6 +14,8 @@ include "attachments.thrift"
 include "vendors.thrift"
 include "components.thrift"
 include "sw360.thrift"
+include "commonobligations.thrift"
```

Comment:
why is this and the additions in line 18 and 37 necessary? (disclaimer: i am no thrift expert)

Reply:
this is just remains of another approach. good catch